hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1804,6 +1815,18 @@
       milestone: V2
       tests: []
     }
+    {
+      name: chip_sw_csrng_known_answer_tests
+      desc: '''Verify our ability to run known-answer tests in SW.
+
+            - Configure the software instance with the expected seed
+              (as per the NIST-specified test for CTR_DRBG operation)
+            - Perform several generate operations
```

Comment:
Add a bit more detail, like which class of NIST vector tests (no_reseed, pr_false, pr_true), and how many of the examples will be run. Remove "Perform several generate operations", but rather follow the exact sequence as outlined by each vector test.